flutter_kernel_transformers/lib/track_widget_constructor_locations.dart

```diff
@@ -149,26 +149,26 @@ class _WidgetCallSiteTransformer extends Transformer {
     Location location, {
     String name,
     ListLiteral parameterLocations,
-    bool showFile: true,
+    bool showFile = true,
   }) {
     final List<NamedExpression> arguments = <NamedExpression>[
-      new NamedExpression('line', new IntLiteral(location.line)),
-      new NamedExpression('column', new IntLiteral(location.column)),
+      NamedExpression('line', IntLiteral(location.line)),
+      NamedExpression('column', IntLiteral(location.column)),
     ];
     if (showFile) {
-      arguments.add(new NamedExpression(
-          'file', new StringLiteral(location.file.toString())));
+      arguments.add( NamedExpression(
+          'file', StringLiteral(location.file.toString())));
     }
     if (name != null) {
-      arguments.add(new NamedExpression('name', new StringLiteral(name)));
+      arguments.add( NamedExpression('name', StringLiteral(name)));
```

Comment:
remove space after parenthesis (here and at several place in the diff)

Reply:
Done